jupyterlab_gitsync/README.md

```diff
@@ -0,0 +1,20 @@
+# JupyterLab Git Sync Extension
+
+JupyterLab extension project
+
+## Development
+
+1. Follow the [instructions](../#Development) from the root of the repository.
```

Comment:
Maybe not, although I think the development section is helpful for future use. This section was automatically generated by cookiecutter, but I removed the gcloud authorization since it isn't used in my project.

Reply:
I think there was a misunderstanding here - I was referring to the Development section, but it looks like you removed the section after this one, which is fine, but the reason why I asked if it's necessary is because 1) the instructions page being linked is a relative link. If you try to click on it, you will get a 404 page not found. 2) this is an intern project, so it's not really a priority to include such a section, but if you want to, then that'd be great. However, it shouldn't be left as is.

Reply:
I just fixed the link so that it links to the correct page for the development instructions.

Reply:
great thanks